addons/im_livechat/static/src/core/common/thread_model_patch.js

```diff
@@ -31,7 +31,12 @@ patch(Thread.prototype, {
             /** @this {import("models").Thread} */
             onUpdate() {
                 if (this.open_chat_window) {
-                    this.openChatWindow({ focus: true });
+                    const cw = this.openChatWindow();
+                    if (this.store.env.services.ui.isSmall) {
+                        cw.fold();
```

Comment:
Shouldn't it only fold if it wasn't already open?

Reply:
Problem is that the recovery from local storage can be slower than receiving `open_chat_window: true` from server, so we cannot tell whether there's an open chat window.

Because of this, if I put this code, the livechat will frequently assume there's no chat window open and always open the chat window in full screen on mobile.

Reply:
It's always possible to check if there is one, if the chat window exists and is in chathub opened.

We would not like to get in a situation where we receive a late `open_chat_window` (bus? network?) and fold an existing chat window in mobile out of nowhere.

Reply:
> It's always possible to check if there is one, if the chat window exists and is in chathub opened.

I attempted that check but locally this didn't work and always assumed the chat window was not open. At a glance this came from the handling of `open_chat_window: true` happening before the recovery of chat windows in local storage. I haven't investigated further why this is the case, but this makes the check not reliable :(

Reply:
It's not supposed to happen at init. It's just a suggestion to make the core more robust.

Reply:
I made it handle `open_chat_window` when the chat hub has been loaded at least once.